### PR TITLE
HTML assignment week 1

### PR DESCRIPTION
**Pull request by lalitchoudhary09**
@anilrao316

### CODE REVIEW

**Comment by lalitchoudhary09** [comment]
working screesnhot of assignment
screenshot added

File: Week-1/HTML Assignment1/index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
+<html>
+  <head>
+	<link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+    <style>	
+		body {
+			font-family: 'Lato';
+		}
+      .bottomLabel {
+        color: white;
+      }
```
Comment (anilrao316) [inline Week-1/HTML Assignment1/index.html]: move css to external file

File: Week-1/HTML Assignment1/index.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        <div style = "display: inline-block; padding-top: 20px">  
+        <label class="bottomLabel">If it Ain't Love</label>
+        </br>
+        <img class="bottomIcon1" src="icon/backward-solid.svg" />
+        <img class="bottomIcon" src="icon/pause-solid.svg" />
+        <img class="bottomIcon" src="icon/forward-solid.svg" />
+        <img class="bottomIcon" src="icon/rev.svg" />
+        <img class="bottomIcon" src="icon/random-solid.svg" />
+      </br>
+        <label class="bottomLabel">Catson Derulo</label>
```
Comment (anilrao316) [inline Week-1/HTML Assignment1/index.html]: Plese follow BEM naming conventions for CSS u can refer this link http://getbem.com/naming/

File: Week-1/HTML Assignment1/index.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+      </table>
+    </main> 
+     <footer class="footerSection">
+      <div style="background-color: #6082b6; bottom: 1px; left: 5px">
+        <div style="float: left;">
+          <img
+            src="https://vif1g.csb.app/src/assets/cattyboard.jpg"
+            width="80"
+            height="80"
+            style="padding: 10px 10px 5px;"
```
Comment (anilrao316) [inline Week-1/HTML Assignment1/index.html]: do not use inline styles move it to external css file. Also i see it being used above as well ,do changes  there as well

File: Week-1/HTML Assignment1/index.html [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+        <tr>
+            <td>4.Work From Home</td>
+            <td>Cat Harmony feat Colla</td>
+            <td>3.34</td>
+            <td><label class="CATTY">CATTY CLOUD SYNC</label></td>
+        </tr>
+      </table>
+    </main> 
+     <footer class="footerSection">
+      <div style="background-color: #6082b6; bottom: 1px; left: 5px">
```
Comment (anilrao316) [inline Week-1/HTML Assignment1/index.html:130]: same here

File: Week-1/HTML Assignment1/index.html [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+      .bottomLabel {
+        color: white;
+      }
+      .bottomIcon {
+        padding-right: 100px;
+        height: 12px;
+        width: 12px;
+      }
+      .bottomIcon1 {
+        padding-left: 1000px;
```
Comment (anilrao316) [inline Week-1/HTML Assignment1/index.html]: 1000px? what is this for ?seems to be huge value

File: Week-1/HTML Assignment1/index.html [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+      {
+        color:white;
+        font-size: small;
+      }
+      .iconWithLabel{
+        padding-right: 20px;
+        height:12px;
+        width:12px;
+        color: white
+       }
```
Comment (anilrao316) [inline Week-1/HTML Assignment1/index.html]: Also install prettier plugin in your visual studio for formatting the code

**Comment by anilrao316** [comment]
Always better to have mutliple commits. can be followed in the upcoming assignments

File: Week-1/HTML Assignment1/index.html (hunk 1 above)
Comment (lalitchoudhary09) [reply Week-1/HTML Assignment1/index.html]: moved all internal and inline css to external

File: Week-1/HTML Assignment1/index.html (hunk 2 above)
Comment (lalitchoudhary09) [reply Week-1/HTML Assignment1/index.html]: followed BEM naming convention

File: Week-1/HTML Assignment1/index.html (hunk 3 above)
Comment (lalitchoudhary09) [reply Week-1/HTML Assignment1/index.html]: moved all css into external

File: Week-1/HTML Assignment1/index.html (hunk 5 above)
Comment (lalitchoudhary09) [reply Week-1/HTML Assignment1/index.html]: updated with new units
 padding-left: 65em;

File: Week-1/HTML Assignment1/index.html (hunk 6 above)
Comment (lalitchoudhary09) [reply Week-1/HTML Assignment1/index.html]: install and applied prettier code formatter

**Comment by lalitchoudhary09** [comment]
@anilrao316 : Can I go ahead and merge this changes into master?

**Pull request merged**